Gemfile.lock

```diff
@@ -315,76 +316,78 @@ GEM
       momentjs-rails (>= 2.8.1)
     brakeman (5.4.1)
     builder (3.2.4)
-    bundler-audit (0.9.0.1)
+    bundler-audit (0.9.1)
       bundler (>= 1.2.0, < 3)
       thor (~> 1.0)
     byebug (8.2.5)
-    chef-utils (18.1.29)
+    chef-utils (18.4.2)
       concurrent-ruby
-    coderay (1.1.1)
+    coderay (1.1.3)
     commonmarker (0.23.10)
-    concurrent-ruby (1.2.2)
-    connection_pool (2.2.1)
-    crack (0.4.3)
-      safe_yaml (~> 1.0.0)
+    concurrent-ruby (1.2.3)
+    connection_pool (2.4.1)
+    crack (0.4.6)
+      bigdecimal
+      rexml
     crass (1.0.6)
-    dalli (3.2.3)
+    dalli (3.2.7)
+      base64
     ddtrace (0.45.0)
       msgpack
     debug_inspector (0.0.3)
     diffy (3.4.2)
-    dogstatsd-ruby (3.0.0)
-    domain_name (0.5.20190701)
-      unf (>= 0.0.5, < 1.0.0)
-    doorkeeper (5.6.6)
+    dogstatsd-ruby (3.3.0)
+    domain_name (0.6.20240107)
```

Comment:
### [domain_name](https://github.com/knu/ruby-domain_name)

**Minor** version upgrade :chart_with_upwards_trend::large_orange_diamond: 0.5.20190701 → 0.6.20240107

[_~~change-log~~, [source-code](https://github.com/knu/ruby-domain_name), [gem-diff](https://my.diffend.io/gems/domain_name/0.5.20190701/0.6.20240107)_]





<details>
<summary>Commits</summary>

A change of **14** commits. See the full changes on [the compare page](https://github.com/knu/ruby-domain_name/compare/v0.5.20190701...v0.6.20240107).

These are the first 10 commits:
- (17ae27b) [Avoid use of `<<~WORD` as it causes syntax error on older ru…](https://github.com/knu/ruby-domain_name/commit/17ae27bbe6c5e21ce97b9137537f97f8e9329e4d)
- (8d1c3e6) [OK, 1.8.7 is still supported](https://github.com/knu/ruby-domain_name/commit/8d1c3e66ce4d904f854ab9ddf42193f8d543c744)
- (c64a590) [Add ruby 2.4-2.6 and retire 2.0-2.3 and rbx-2](https://github.com/knu/ruby-domain_name/commit/c64a59027939aa34e1f5f0efc5cb654d73ccb966)
- (4d08512) [Require Ruby >=2.7 and drop the dependency on unf](https://github.com/knu/ruby-domain_name/commit/4d08512da35ce1edb156ef70ed4781b9664a0f26)
- (88b13c2) [Migrate the CI to GitHub Actions](https://github.com/knu/ruby-domain_name/commit/88b13c2d117bafccf07918338466d3807a0b2370)
- (9701b77) [Bump the minor version to 0.6](https://github.com/knu/ruby-domain_name/commit/9701b77ad2c464c4a772629302fb1f9bccadaac6)
- (c2c0e97) [Update the changelog](https://github.com/knu/ruby-domain_name/commit/c2c0e978e726856e60d1e132f34df6a3452381ef)
- (d64fd23) [Update the eTLD database to 2023-11-09 00:10:10 UTC.](https://github.com/knu/ruby-domain_name/commit/d64fd2307f2796fd180b908666a782857b941525)
- (567c4cd) [allows DomainName.normalize to accept ASCII-only non-unicode…](https://github.com/knu/ruby-domain_name/commit/567c4cd385d2535626ce8d7d190b22e07772564e)
- (e60aae9) [Merge pull request #27 from tisba/fix/issue-23](https://github.com/knu/ruby-domain_name/commit/e60aae96a33471f93d588f6df10d62293721d783)


</details>